week-1/L-functions-nested/excercise.js

```diff
@@ -0,0 +1,14 @@
+function createMessage(message) {
```

Comment:
Great! Your program accomplishes what it needs to do, and the answers are correct.  
HOWEVER... given that "You should have one function that calculates the percentage, and one function that creates a message" - I would argue: 
1) the "creation" of the message should include the concatenation of its parts.
2) Your function to "calculate the percentage" does too much.  It doesn't only calculate a percentage, it calculates TWO percentages, creates and displays messages about them.
Can you change your program to make use of a function that only calculates a percentage based on the numbers given to it?

Reply:
A very minor point: watch out spelling 'exercise'.  It's not 'excercise' which you've used in a few places.